packages/components/src/components/Cell/CellGroup.tsx

```diff
@@ -62,10 +62,10 @@ export const CellGroup = ({
     );
 };
 
-const themeStyles = EDSStyleSheet.create(theme => ({
+const themeStyles = EDSStyleSheet.create((theme, props: CellGroupProps) => ({
```

Comment:
I think it would be wise to create a separate `CellGroupStyleProps` that you can control separately from the `CellGroupProps`. If we were to add a new required prop to the `CellGroupProps`, it would now automatically require this new prop in the style object.

Reply:
Updated CellGroup now. Was it something like this you had in mind ?